tensorflow/python/data/experimental/kernel_tests/parallel_interleave_test.py

```diff
@@ -22,24 +22,24 @@
 import threading
 import time
 
+from absl.testing import parameterized
 import numpy as np
 from six.moves import zip_longest
 
 from tensorflow.python.data.experimental.ops import interleave_ops
 from tensorflow.python.data.kernel_tests import test_base
 from tensorflow.python.data.ops import dataset_ops
+from tensorflow.python.framework import combinations
 from tensorflow.python.framework import dtypes
 from tensorflow.python.framework import errors
 from tensorflow.python.framework import sparse_tensor
-from tensorflow.python.framework import test_util
 from tensorflow.python.ops import math_ops
 from tensorflow.python.ops import script_ops
 from tensorflow.python.ops import sparse_ops
 from tensorflow.python.platform import test
 
 
-@test_util.run_all_in_graph_and_eager_modes
-class ParallelInterleaveTest(test_base.DatasetTestBase):
+class ParallelInterleaveTest(test_base.DatasetTestBase, parameterized.TestCase):
```

Comment:
The tests in this class could be parametrized similar to [interleave_test](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/data/kernel_tests/interleave_test.py). I would defer that to a separate PR so maybe just add a TODO here.

Reply:
Sounds good.